x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression/TypeResolutions.java

```diff
@@ -170,7 +170,7 @@ public static TypeResolution isType(
             );
     }
 
-    private static String acceptedTypesForErrorMsg(String... acceptedTypes) {
+    public static String acceptedTypesForErrorMsg(String... acceptedTypes) {
```

Comment:
I imagine we should move the utility function into the type rather than make this public.

Reply:
done